lib/storage/metadata/bucketclient/RecordLogProxy.js

```diff
@@ -0,0 +1,108 @@
+'use strict'; // eslint-disable-line
+
+const stream = require('stream');
+
+const errors = require('../../../errors');
+const jsutil = require('../../../jsutil');
+
+class ListRecordStream extends stream.Transform {
+    constructor(logger) {
+        super({ objectMode: true });
+        this.logger = logger;
+    }
+
+    _transform(itemObj, encoding, callback) {
+        itemObj.entries.forEach(entry => {
+            // eslint-disable-next-line no-param-reassign
+            entry.type = entry.type || 'put';
+        });
+        this.logger.debug('processing log item', { itemObj });
+        this.push(itemObj);
+        callback();
```

Comment:
This should be ok, it's called by the node engine internally and the doc examples just call the callback directly as a way to notify that we processed the items.